From: Christian Melki <christian.melki@t2data.com>
Date: Wed, 15 Nov 2023 08:52:11 +0100
Subject: [PATCH] copy-firmware.sh: Don't depend on exotic binaries that aren't
 needed.

We don't need this to do the install. I don't know why
there is a hard requirement on it.

Signed-off-by: Christian Melki <christian.melki@t2data.com>
---
 copy-firmware.sh | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/copy-firmware.sh b/copy-firmware.sh
index f9b1f0ffc1c3..cca3913bd944 100755
--- a/copy-firmware.sh
+++ b/copy-firmware.sh
@@ -69,11 +69,6 @@ if [ -z "$destdir" ]; then
 	exit 1
 fi
 
-if ! which rdfind 2>/dev/null >/dev/null; then
-	echo "ERROR: rdfind is not installed"
-	exit 1
-fi
-
 # shellcheck disable=SC2162 # file/folder name can include escaped symbols
 grep -E '^(RawFile|File):' WHENCE | sed -E -e 's/^(RawFile|File): */\1 /;s/"//g' | while read k f; do
     test -f "$f" || continue
